README.md

```
@@ -26,37 +26,74 @@ Start quicker with a pre-installed UI Framework.
-1. Click the **Use this template** button in GitHub and follow the on-screen instructions to **Create a new repository**.
+[nvm](https://github.com/nvm-sh/nvm) or [mise](https://mise.jdx.dev/) users:
+`nvm use` / `mise install` provides the pinned versions.
+
+### One-command setup (local PHP + SQLite, no Docker)
 
-2. Clone the repository locally.
+1. Create your repository from this template (or clone it), then from
+   the repository root:
 
-   Example: `git clone git@github.com:druxt/quickstart.git`
+   ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)